firewood/src/merkle.rs

```diff
@@ -1723,7 +1729,7 @@ mod tests {
 
     fn branch(value: Vec<u8>, encoded_child: Option<Vec<u8>>) -> Node {
         let children = Default::default();
-        let value = Some(Data(value));
+        let value = Some(Data(value)).filter(|data| !data.is_empty());
```

Comment:
This is because the test has branch node with [empty value](https://github.com/ava-labs/firewood/pull/425/files#diff-6a6e300f3d9050248dbafa719531f14eabecc1bfac2169bb2527c81ee5474ecaL1745). Want to cover it correctly.